doc/en/fixture.rst

```diff
@@ -180,8 +180,8 @@ function (in or below the directory where ``conftest.py`` is located)::
     def test_ehlo(smtp):
         response, msg = smtp.ehlo()
         assert response == 250
-        assert "smtp.gmail.com" in str(msg, 'ascii')
```

Comment:
The `str(msg, 'ascii')` idiom because of python3, which I think at one point was how the docs were being generated. (`msg` is `bytes`)